### PR TITLE
TFE v202103-1 release updates

### PR DESCRIPTION
**Pull request by p0pr0ck5**

### CODE REVIEW

File: content/source/docs/enterprise/install/automating-the-installer.html.md [hunk 1]
```diff
@@ -200,7 +201,7 @@ If you have chosen `external` for `production_type`, the following settings appl
 - `pg_password` — (Required) The password for the PostgreSQL user.
 - `pg_netloc` — (Required) The hostname and port of the target PostgreSQL server, in the format `hostname:port`.
 - `pg_dbname` — (Required) The database name.
-- `pg_extra_params` — (Optional) Parameter keywords of the form `param1=value1&param2=value2` to support additional options that may be necessary for your specific PostgreSQL server.  Allowed values are [documented on the PostgreSQL site](https://www.postgresql.org/docs/9.4/static/libpq-connect.html#LIBPQ-PARAMKEYWORDS).  An additional restriction on the `sslmode` parameter is that only the `require`, `verify-full`, `verify-ca`, and `disable` values are allowed.
+- `pg_extra_params` — (Optional) Parameter keywords of the form `param1=value1&param2=value2` to support additional options that may be necessary for your specific PostgreSQL server.  Allowed values are [documented on the PostgreSQL site](https://www.postgresql.org/docs/9.4/libpq-connect.html#LIBPQ-PARAMKEYWORDS).  An additional restriction on the `sslmode` parameter is that only the `require`, `verify-full`, `verify-ca`, and `disable` values are allowed.
```
Comment (aaron-lane) [inline content/source/docs/enterprise/install/automating-the-installer.html.md:204]: Should we refer to a version of the documentation more recent than 9.4?

Comment (p0pr0ck5) [reply content/source/docs/enterprise/install/automating-the-installer.html.md]: Makes sense, considering we're bumping the included version of postgres. Updated!

**Review by sudomateo — APPROVED** [review]

**Review by rogeruiz — APPROVED** [review]
lgtm 🌈

**Comment by rogeruiz** [comment]
Explanation of the failed CI on this PR.

```
Results:
=== Found broken links! ===
Fix before merging... or if they're not really broken, explain why.

content/source/docs/enterprise/install/installer.html.md
  - Broken link: https://install.terraform.io/airgap/latest.tar.gz
    [301 Moved Permanently] https://install.terraform.io/airgap/latest.tar.gz
```

```
curl -I -L https://install.terraform.io/airgap/latest.tar.gz
HTTP/2 301 
date: Tue, 23 Mar 2021 14:11:26 GMT
content-type: application/json
content-length: 0
location: https://s3.amazonaws.com/replicated-airgap-work/replicated.tar.gz
x-amzn-requestid: a27b3263-ffea-4b0c-9ace-e5069fa4abce
x-amz-apigw-id: cpOKNHWZoAMF61A=
x-amzn-trace-id: Root=1-6059f70e-0f4630426e870ff967001b54;Sampled=0

HTTP/1.1 200 OK
x-amz-id-2: j4RxoejMrcD4QnyERVqp6qKbhtqXxGT6GxrcIATtfk8vi4FZdjQ08U+FQSKslPbPgX8o9jmxyko=
x-amz-request-id: 4ZDB17S5H1TP148X
Date: Tue, 23 Mar 2021 14:11:27 GMT
Last-Modified: Mon, 22 Feb 2021 17:41:49 GMT
ETag: "5be46c0b454899603e88ce31552f40b4"
Accept-Ranges: bytes
Content-Type: binary/octet-stream
Content-Length: 990580310
Server: AmazonS3
```

**Pull request merged**